### PR TITLE
Update listed modules/scripts.

### PR DESCRIPTION
**Pull request by heyitsbench**
Adds arena 3v3 solo queue and replay modules, synchronizes the two places where modules are listed, and aligns the names to match their respective repository titles.

### CODE REVIEW

**Comment by Helias** [comment]
we have to add https://github.com/gozzim/mod-npc-spectator/ and mod-1v1-arena too

**Comment by heyitsbench** [comment]
> we have to add [mod-npc-spectator] and mod-1v1-arena too

Done.

File: .github/CC_SERVER_INFO.md [hunk 1]
```diff
@@ -4,9 +4,13 @@ Ubuntu 20.04
 
 ##### MODULES:
 
+- [mod-1v1-arena](https://github.com/azerothcore/mod-1v1-arena)
 - [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
 - [mod-anticheat](https://github.com/azerothcore/mod-anticheat)
+- [mod-arena-3v3-solo-queue](https://github.com/azerothcore/mod-arena-3v3-solo-queue)
+- [mod-arena-replay](https://github.com/azerothcore/mod-arena-replay)
 - [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
+- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
```
Comment (Nyeriah) [inline .github/CC_SERVER_INFO.md:13]: This is wrong, we are using the forked module at @ AC

Comment (heyitsbench) [reply .github/CC_SERVER_INFO.md]: Done.

**Comment by heyitsbench** [comment]
Maybe also want to include the changes for PTR somewhere for users to be able to grab and contribute to if they want. (E.g. template module and PTR command query repo)

**Comment by Helias** [comment]
> Maybe also want to include the changes for PTR somewhere for users to be able to grab and contribute to if they want. (E.g. template module and PTR command query repo)

PTR is "almost identical" to CC live, not sure if this is necessary, we could maybe show some info about it and the AC-related version, but it's more-or-less the same as CC live.

**Review by Helias — APPROVED** [review]

**Pull request merged**